### PR TITLE
Setup FastAPI APP

### PR DESCRIPTION
**Pull request by abhishek12tri**
Setup API APP
Added in Readme
Removed unwanted

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,8 +49,14 @@ pipenv install -r requirements.txt
 ## To Activate env
 pipenv shell
 
-## To run 
-uvicorn  main:app --reload 
+## To run in localhost
+python3 main.py localhost
```
Comment (abhishekratnam) [inline README.md]: python3 isn't required, make a venv then use it locally. i.e python file_to run

**Comment by abhishekratnam** [comment]
Please take the latest pull, if haven't taken earlier.

File: README.md (hunk 1 above)
Comment (abhishekratnam) [reply README.md]: python -m venv venv/

File: api/stocks.py [hunk 2]
```diff
@@ -1,22 +1,23 @@
+"""
+Author: Ratnam, Abhi
```
Comment (swatcherspace) [inline api/stocks.py:2]: Use git-lens, we don't really need the author's name.

File: routes/api.py [hunk 3]
```diff
@@ -1,6 +1,13 @@
...
+"""
 from fastapi import APIRouter
-from api.stocks import stock
+from api.stocks import route
 
 router = APIRouter()
-router.include_router(stock.router)
 
+
+"""Add stock API router"""
```
Comment (swatcherspace) [inline routes/api.py:12]: Use doc-string only for documentation purpose, else for a normal comment use #

**Pull request closed without merging**